### PR TITLE
Trying out a landing page for Converters

### PR DESCRIPTION
**Pull request by bijington**
@davidbritch @brminnick @jfversluis

Just trying out a landing page for converters that links over to the .NET MAUI docs. The links in the table currently link to our written docs but I suspect these could quite easily link out to the API docs instead. What do you think?

### CODE REVIEW

**Comment by PRMerger4** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by davidbritch** [comment]
@bijington I like it :)

My only minor comment is there's no text between the H1 and the H2, and there should be. Alternatively, I'd dump the H2.

**Comment by bijington** [comment]
> @bijington I like it :)
> 
> My only minor comment is there's no text between the H1 and the H2, and there should be. Alternatively, I'd dump the H2.

Good point! I can drop the first H2.

Out of interest does this TOC change mean that when a user clicks on Converters they get to see this new index page? Of course in the long run we probably won't need the child items (specific converters)

File: docs/maui/TOC.yml [hunk 1]
```diff
@@ -11,6 +11,8 @@ items:
       href: behaviors/maximum-length-reached-behavior.md
 - name: "Converters"
   items:
+    - name: Converters
```
Comment (davidbritch) [inline docs/maui/TOC.yml:14]: I'd recommend calling it Overview.

**Comment by davidbritch** [comment]
> Out of interest does this TOC change mean that when a user clicks on Converters they get to see this new index page? Of course in the long run we probably won't need the child items (specific converters)

Yes. If you dump the child items then there's also no need to have an expandable Converters node e.g. 

Converters
   Overview

would just become:

Converters

**Comment by PRMerger5** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger14** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by PRMerger9** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/converters/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+For further information on Converters please refer to the [.NET MAUI documentation](/dotnet/maui/fundamentals/data-binding/converters).
+
+## .NET MAUI Community Toolkit Converters
+
+The .NET MAUI Community Toolkit provides a collection of pre-built, reusable converters to make developers lives easier. Here are the converters provided by the toolkit:
+
+| Converter | Description |
+| --------- | ----------- |
+| [`EqualConverter`](equal-converter.md) | The `EqualConverter` is a one way converter that returns a `bool` indicating whether the binding value is equal to another specified value. |
```
Comment (jfversluis) [inline docs/maui/converters/index.md:20]: OK last thing I think, not all converters that are added now are in this table

Comment (bijington) [reply docs/maui/converters/index.md:20]: Pesky other Shaun submitting PRs to hinder this progress. Good catch thanks

**Pull request merged**